.scripts/prerequisites.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
```

Comment:
Please add this file to the xcode group, so that it is visible inside the project.

Reply:
Done in 323259f